applications/nvidia_nim/nvidia_nim_imaging/CMakeLists.txt

```diff
@@ -0,0 +1,85 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.20)
+project(nvidia_nim_imaging LANGUAGES NONE)
+
+# # Install ClaraViz
+
+# # we don't need the gRPC interface, OpenH264 and the examples
+# set(CLARA_VIZ_WITH_GRPC OFF CACHE INTERNAL "")
+# set(CLARA_VIZ_WITH_OPENH264 OFF CACHE INTERNAL "")
+# set(CLARA_VIZ_WITH_EXAMPLES OFF CACHE INTERNAL "")
+
+# # we are downloading large content, it's better to see the progress
+# set(FETCHCONTENT_QUIET OFF)
+
+# # Set CMP0135 policy to NEW to use time of extraction for files extracted by
+# # FetchContent/ExternalProject_Add.
+# # This ensures that anything that depends on the extracted contents will be rebuilt whenever
+# # the URL changes.
+# if(POLICY CMP0135)
+#   cmake_policy(SET CMP0135 NEW)
+#   set(CMAKE_POLICY_DEFAULT_CMP0135 NEW)
+# endif()
+
+# # set CMP0148 policy to OLD to allow FindCUDA to be used by ClaraViz
+# if(POLICY CMP0146)
+#   cmake_policy(SET CMP0146 OLD)
+#   set(CMAKE_POLICY_DEFAULT_CMP0146 OLD)
+# endif()
+
+# set(_clara_viz_version "0.3.1")
+# include(FetchContent)
+# FetchContent_Declare(
+#     ClaraViz
+#     URL https://github.com/NVIDIA/clara-viz/archive/refs/tags/v${_clara_viz_version}.tar.gz
+#     URL_MD5 8cc1a29481df378a9413bfe1c3083de4
+#     )
+
+# # enable CMP0077 to allow overwriting option() statements in FetchContent sub-projects
+# cmake_policy(SET CMP0077 NEW)
+
+# FetchContent_GetProperties(ClaraViz)
+# if(NOT claraviz_POPULATED)
+#     FetchContent_Populate(ClaraViz)
+#     set(CLARA_VIZ_PUBLIC_CMAKE_TOOLS_DIR "${claraviz_SOURCE_DIR}/cmake")
+#     add_subdirectory(${claraviz_SOURCE_DIR}/thirdparty ${claraviz_BINARY_DIR}/thirdpary)
+#     add_subdirectory(${claraviz_SOURCE_DIR}/src ${claraviz_BINARY_DIR}/src)
+# endif()
+# set(FETCHCONTENT_QUIET ON)
+
+# set(CLARA_VIZ_PUBLIC_CMAKE_TOOLS_DIR "${claraviz_SOURCE_DIR}/cmake")
+
+# find_package(clara_viz_renderer REQUIRED HINTS ${claraviz_SOURCE_DIR}/cmake)
```

Comment:
Commented out code, please remove.

Reply:
Removed. Thank you @AndreasHeumann. So we do not need to include ClaraViz at all correct?

Reply:
Correct, ClaraViz is a dependency of the volume renderer operator, no need to include it in the app.